gemp-lotr/gemp-lotr-async/src/main/web/src/updated/jCard/modules/image-resolution/data/players-committee/2021/index.ts

```diff
@@ -0,0 +1,18 @@
+import { Override } from "../../../types/override.interface";
+import { aprilErrata } from "./april-errata.list";
+import { decemberErrata } from "./december-errata.list";
+import { holidayPromos } from "./holiday-promos.list";
+import { leaguePromos } from "./league-promos.list";
+import { mayErrata } from "./may-errata.list";
+import { vpack1 } from "./vpack1.list";
+import { vsetOneShadowOfThePast } from "./vset-1-a-shadow-of-the-past.list";
+
+export const updatesFrom2021: Override[] = [
```

Comment:
this pattern is called "barreling" in JavaScript. Normally in an `index.js` file you import all the other files in a given folder and re-export them from this single place. That gives you just one file you import from as opposed to all the various files in this folder (like we did on 1-8).

This is obviously a bit different cuz we are taking all those imports and turning those many arrays into one single array.

By doing this at the year level we limit the number of concats we do at the "global' level and let ti all roll up.